EvenSum.kt

```diff
@@ -0,0 +1,20 @@
+class EvenSum {
+    fun evenSum(array: Array<Int>) : Int
+    {
+        var i:Int = 0
+        var sum:Int = 0
+        for(i in 0..array.size-1 step 1)
+        {
+            println(array[i])
+        }
+
+        for(i in 0..array.size-1 step 1)

```

Comment:
Most modern languages have nicer ways of iterating over arrays, such is the case with kotlin, can you look into an alternative for this for index ?

Reply:
One solution i found was this:  array.forEach { if (array[it]%2==0) sum+= array[it] }